docs/Public_API.rst

```diff
@@ -1123,12 +1123,22 @@ A JSON payload is required to emit your own metrics with the metric value and th
 
         "name": "<CUSTOM METRIC NAME>",
 
-        "value": <CUSTOM METRIC VALUE>
+        "value": <CUSTOM METRIC VALUE>,
+
+        "unit": "<CUSTOM METRIC UNIT>"
 
       }]
 
   }
 
+<INSTANCE INDEX> is the index of current application instance which would be fetched from environment variable `CF_INSTANCE_INDEX`
+
+<CUSTOM METRIC NAME> is the name of the emit metric which must be equal to the metric name that you define in the policy file. 
```

Comment:
1 define->defined?
2 there will be no file if user define policy in ui? So just policy is ok?
3 now in app_metric and appinstancemetrics table the metric_name column is varchar(100), so the length <CUSTOM METRIC NAME> should be <=100;

Reply:
1.`define` should be fine here as the present tense. 
2.  I didn't get your point here ...